nemo/collections/asr/models/ctc_models.py

```diff
@@ -104,7 +104,7 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
         self.setup_optimization_flags()
 
         # setting up interCTC loss (from InterCTCMixin)
-        self.setup_interctc()
+        self.setup_interctc('decoder', 'loss', '_wer')
```

Comment:
Can you use keyword args here ? Kinda hard to tell what is input to this function